### PR TITLE
Create Event Api

### LINKED ISSUE #9: Implement schema and REST API for the Event model.
- Write a DDL (Data Definition Language) that has the event's schema's definition.
- Write a REST API that does CRUD (Cread Read Update Delete) operations on the event model.
- Event model [reference](https://github.com/GDSC-ASU/website/blob/main/src/lib/models/Event.ts)

### PR DESCRIPTION
**Pull request by RemasR**

### CODE REVIEW

File: src/lib/utils/requests/EventRequests.ts [hunk 1]
```diff
@@ -1,38 +1,32 @@
...
+	  const events = await Event.findAll();
+	  return events.map((event) => event.toJSON());
 	}
+	public static async createEvent(name: string, description:string, link: string, imagePath: string, altImagePath: string) {
+  const newEvent = await Event.create({
+	name,
+	description,
+	link,
+	imagePath,
+	altImagePath
```
Comment (RemasR) [inline src/lib/utils/requests/EventRequests.ts]: send 1 Event param instead of sending 6 params

File: src/lib/utils/requests/EventRequests.ts [hunk 2]
```diff
@@ -1,38 +1,32 @@
...
+});
+	}
+public static async updateEvent(id: string,name: string, description:string, link: string, imagePath: string, altImagePath: string){
+	const events = await Event.findByPk(id); 
+	await events.update({
+		name,
+		description,
+		link,
+		imagePath,
+		altImagePath
```
Comment (RemasR) [inline src/lib/utils/requests/EventRequests.ts]: same here

File: src/lib/data/gdsc.db
Comment (mbaraa) [inline src/lib/data/gdsc.db:1]: @RemasMehyar 
remove this file, and add it to .gitignore

File: src/lib/data/gdscDB.ts [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import { Sequelize } from "sequelize-typescript";
+import { DataType } from "sequelize-typescript";
+
+export const sequelize = new Sequelize({
+  dialect: "sqlite",
+  storage: "$lib/data/gdsc.db",
```
Comment (mbaraa) [inline src/lib/data/gdscDB.ts]: @RemasMehyar 
remove this static path, instead utilize the environment variable `VITE_SQLITE_FILE_PATH`, and add the static path to `.env.example` file.

File: src/lib/utils/requests/EventRequests.ts (hunk 1 above)
Comment (mbaraa) [reply src/lib/utils/requests/EventRequests.ts]: can you clarify even more?

File: src/lib/utils/requests/EventRequests.ts (hunk 2 above)
Comment (mbaraa) [reply src/lib/utils/requests/EventRequests.ts]: same here

File: package.json [hunk 4]
```diff
@@ -27,6 +28,15 @@
     },
     "type": "module",
     "dependencies": {
-        "@sveltejs/adapter-node": "^1.0.0-next.96"
-    }
```
Comment (RemasR) [inline package.json:31]: test

**Review by mbaraa — CHANGES_REQUESTED** [review]
check the change request comments

**Review by mbaraa — APPROVED** [review]
good job!

**Pull request merged**